packages/components/src/engine/metadata.ts

```diff
@@ -19,34 +82,25 @@ export const getMetadata = (props: IsomerPageSchemaType) => {
       icon: faviconUrl,
       shortcut: faviconUrl,
     },
+    openGraph: {
+      title: props.page.title,
+      description: getMetaDescription(props),
+      url: fullUrl,
+      siteName: props.site.siteName,
+      type: props.layout === "article" ? "article" : "website",
+      images: !!metaImageUrl
+        ? [
+            {
+              url: metaImageUrl,
+            },
+          ]
+        : undefined,
+    },
```

Comment:
question: what's `openGraph` here and what does it do?

Reply:
Opengraph is the protocol for sharing links on social media (docs: https://ogp.me/). The object's shape is taken from here: https://nextjs.org/docs/app/api-reference/functions/generate-metadata#opengraph